### DISCUSSION TITLE
track injection virus volume

### DISCUSSION THREAD

**Discussion by mekhlakapoor** [AllenNeuralDynamics/aind-data-schema#994] [Ideas]
the LAS sharepoint list tracks the volume of virus that's injected during RO injections. This seems like something we should track in our ViralMaterial model.

**Comment by saskiad** [comment]
This is already tracked in the injection procedure model

**Reply by mekhlakapoor** [reply]
The request is for "virus_volume" , which is different than the injection_volume. If that already exists, can you point me to that?

**Reply by dbirman** [reply]
@mekhlakapoor I'm going to close this since the solution is that ViralMaterial has the titer and RetroOrbitalInjection has that volume, so virus_volume = titer * injection_volume after accounting for units.
